src/pages/markets/components/helper/_available-platforms.js

```diff
@@ -60,6 +61,7 @@ const AvailablePlatforms = ({
     dtrader,
     dbot,
     smarttrader,
+    derivX,
```

Comment:
```suggestion
    derivx,
```

Reply:
Just so we're consistent with the other platforms. For the other instances as well 👍